app/javascript/dashboard/components/widgets/chart/HorizontalBarChart.js

```diff
@@ -0,0 +1,54 @@
+import { HorizontalBar } from 'vue-chartjs';
+
+const chartOptions = {
+  responsive: true,
+  legend: {
+    display: false,
+  },
+  title: {
+    display: false,
+  },
+  tooltips: {
+    enabled: false,
+  },
+  scales: {
+    xAxes: [
```

Comment:
Is this xAxis or typo ?

Reply:
No, it is xAxes. We are using chart.js 2.9.4.